langchain4j-cloud-sql-postgres/pom.xml

```diff
@@ -30,11 +30,179 @@
             <artifactId>pgvector</artifactId>
             <version>${pgvector-java.version}</version>
         </dependency>
+       <dependency>
+         <groupId>com.google.cloud.sql</groupId>
+        <artifactId>postgres-socket-factory</artifactId>
+        <version>1.21.0</version>
+       </dependency>
+
+       <dependency>
+        <groupId>com.google.oauth-client</groupId>
+        <artifactId>google-oauth-client</artifactId>
+        <version>1.36.0</version>
+        <exclusions>
+            <exclusion>
+            <groupId>com.google.http-client</groupId>
+            <artifactId>google-http-client-gson</artifactId>
+            </exclusion>
+            <exclusion>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            </exclusion>
+                <exclusion>
```

Comment:
nit: Could you run this through a formatter quick? indentation is off in multiple places